frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -305,17 +322,34 @@ export function SurveyResult({ disableEventsTable }: { disableEventsTable?: bool
                                 {question.scale === 10 && (
                                     <>
                                         <div className="text-4xl font-bold">{surveyNPSScore}</div>
-                                        <div className="font-semibold text-muted-alt mb-2">Total NPS Score</div>
-                                        {/* TODO: rework this to show nps scores over time */}
+                                        <div className="font-semibold text-muted-alt mb-2">Latest NPS Score</div>
                                         <SurveyNPSResults survey={survey as Survey} />
                                     </>
                                 )}
+
                                 <RatingQuestionBarChart
                                     key={`survey-q-${i}`}
                                     surveyRatingResults={surveyRatingResults}
                                     surveyRatingResultsReady={surveyRatingResultsReady}
                                     questionIndex={i}
+                                    iteration={survey.current_iteration}
```

Comment:
This is something I'm getting feedback today on, with the customer who filed this bug. 
We have a way to get the NPS results for the current iteration right above this chart, and we retain the "current iteration" view here. 
I dunno if it makes more sense to show all responses since the start of the survey here, or just the ones received in the current iteration. I'll make a call later today.

Reply:
I meant this specifically for non-NPS rating charts, where we have no extra calculations like we do for NPS.